Snippets/Snippets.json

```diff
@@ -24,47 +24,71 @@
       ],
       "description": "Get Azure Sentinel configuration parameters from config.json"
     },
-  
-    "Sentinel - Set up environment for msticpy": {
-      "prefix": ["setup-msticpy"],
+
+    "Sentinel - Set up environment for MSTICPy": {
+      "prefix": ["msticpy-setup"],
       "body": [
         "#####################################################",
-        "# Sentinel - Set up environment for msticpy         #",
+        "# Sentinel - Set up environment for MSTICPy         #",
         "#####################################################",
         "from pathlib import Path",
-        "import os",
-        "import sys",
-        "import warnings",
-        "from IPython.display import display, HTML, Markdown",
+        "from IPython.display import display, HTML",
+        "",
+        "update_nbcheck = (",
```

Comment:
I'm thinking we should delete the nb_check stuff since it doesn't do much any more and is a pain to maintain